Microsoft.Toolkit.Uwp.UI.Media/Effects/Interfaces/IPipelineNode.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Uwp.UI.Media.Pipelines;
+
+namespace Microsoft.Toolkit.Uwp.UI.Media.Effects
+{
+    /// <summary>
+    /// The base <see langword="interface"/> for all the builder effects to be used in a <see cref="PipelineBrush"/>
+    /// </summary>
+    public interface IPipelineNode
```

Comment:
Does it make further sense to separate out things which are just effects (modify inbetween the pipeline) and things which are both (take input and can have further effects)?

Reply:
i.e. I feel like the interfaces should also have the properties (Input/Effects) on them too, no?

Reply:
Just as a note (we discussed this point already), this interface has now been renamed/moved, and the naming issue has been clarified. See comments below for more info.